manifests/c/CiderCollective/Cider/Nightly/1.3.1100/CiderCollective.Cider.Nightly.installer.yaml

```diff
@@ -0,0 +1,17 @@
+# Created using wingetcreate 0.5.0.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: CiderCollective.Cider.Nightly
+PackageVersion: 1.3.1100
+Installers:
+- MinimumOSVersion: 10.0.0.0
+  Architecture: x86

```

Comment:
```suggestion
  Architecture: x64
```

Reply:
bot screenshot of the issue:
![InstallationVerification-x86-Scope_Undefined-Locale_Undefined-ErrorScreenshot_20220308195834](https://user-images.githubusercontent.com/15158490/157725881-2d5c147e-b76f-49e8-881c-a566d4db210f.png)